chrome/content/zotero/xpcom/zotero.js

```diff
@@ -1368,6 +1368,13 @@ Services.scriptloader.loadSubScript("resource://zotero/polyfill.js");
 	this.randomString = function(len, chars) {
 		return Zotero.Utilities.randomString(len, chars);
 	}
+
+	this.nextHtmlId = (prefix = 'id-') => {
+		if (!this._htmlId) {
+			this._htmlId = 1;
+		}
+		return prefix + this._htmlId++;
+	}
```

Comment:
I've been putting things needed for React UI here https://github.com/zotero/zotero/blob/7b77c0cd27c9c5c78ce4f14bb517024ccf72de2f/chrome/content/zotero/components/utils.js

(There's more stuff coming to that file with the tree PR). We may need to figure out a better, more appropriately named place for those in the future, but a function like this should certainly not live on the main Zotero object.